apps/condo/containers/TicketForm/UpdateTicketForm.tsx

```diff
@@ -36,6 +45,41 @@ export const UpdateTicketForm:React.FC<IUpdateTicketForm> = ({ id }) => {
             action={updateAction}
             initialValues={Ticket.convertGQLItemToFormState(obj)}
             organization={organization}
-        />
+        >
+            {({ handleSave, isLoading }) => {
```

Comment:
Why we have two forms for create/update?

Reply:
there may be different logic before / after creation / editing of ticket